stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -449,45 +449,11 @@ diag(A::AbstractVector) = throw(ArgumentError("use diagm instead of diag to cons
 # Dot products and norms
 
 # special cases of norm; note that they don't need to handle isempty(x)
-function generic_normMinusInf(x)
-    (v, s) = iterate(x)::Tuple
-    minabs = norm(v)
-    while true
-        y = iterate(x, s)
-        y === nothing && break
-        (v, s) = y
-        vnorm = norm(v)
-        minabs = ifelse(isnan(minabs) | (minabs < vnorm), minabs, vnorm)
```

Comment:
As a note, this line can now be removed due to https://github.com/JuliaLang/julia/pull/12563 which makes https://github.com/JuliaLang/julia/pull/12564 redundant.